src/systems/multicopter_motor_model/MulticopterMotorModel.cc

```diff
@@ -239,47 +239,72 @@ void MulticopterMotorModel::Configure(const Entity &_entity,
   if (!this->dataPtr->model.Valid(_ecm))
   {
     gzerr << "MulticopterMotorModel plugin should be attached to a model "
-           << "entity. Failed to initialize." << std::endl;
+          << "entity. Failed to initialize." << std::endl;
     return;
   }
 
   auto sdfClone = _sdf->Clone();
 
   this->dataPtr->robotNamespace.clear();
 
-  if (sdfClone->HasElement("robotNamespace"))
+  if (sdfClone->HasElement("robot_namespace"))
+  {
+    this->dataPtr->robotNamespace =
+        sdfClone->Get<std::string>("robot_namespace");
+  }
+  else if (sdfClone->HasElement("robotNamespace"))
   {
     this->dataPtr->robotNamespace =
         sdfClone->Get<std::string>("robotNamespace");
+    gzwarn << "<robotNamespace> is deprecated "
```

Comment:
Per our discussion in the meeting, these warnings should only be added in the `main` branch for Harmonic.